tests/Doctrine/Tests/DBAL/Events/SqliteSessionInitTest.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Doctrine\Tests\DBAL\Events;
+
+use Doctrine\DBAL\Event\ConnectionEventArgs;
+use Doctrine\DBAL\Event\Listeners\SqliteSessionInit;
+use Doctrine\DBAL\Events;
+use Doctrine\Tests\DbalTestCase;
+
+class SqliteSessionInitTest extends DbalTestCase
+{
+    public function testPostConnect(): void
+    {
+        $connectionMock = $this->createMock('Doctrine\DBAL\Connection');
+        $connectionMock->expects($this->once())
+                       ->method('exec')
+                       ->with($this->equalTo('PRAGMA foreign_keys = on'));
```

Comment:
You can omit `$this->equalTo()` in `with()`. If you just use `'PRAGMA foreign_keys = on'` in `with()`, PHPUnit will handle it with an `IsEqual` constraint anyway.